spec/rubocop/cop/style/hash_as_last_array_item_spec.rb

```diff
@@ -54,6 +54,42 @@
       RUBY
     end
 
+    it 'registers an offense and corrects when hash with braces and trailing comma' do
+      expect_offense(<<~RUBY)
+        [1, 2, { one: 1, two: 2, },]
+               ^^^^^^^^^^^^^^^^^^^ Omit the braces around the hash.
+      RUBY
+
+      expect_correction(<<~RUBY)
+        [1, 2,  one: 1, two: 2, ]
+      RUBY
+    end
+
+    it 'registers an offense and corrects when hash with braces and trailing comma and new line' do
+      expect_offense(<<~RUBY)
+        [
+          1,
+          2,
+          {
+          ^ Omit the braces around the hash.
+            one: 1,
+            two: 2,
+          },
+        ]
```

Comment:
Should there be a testcase for this scenario as well?
```ruby 
 [
    1,
    2,
    {
      one: 1,
      two: 2,
    },
    3,
    4
]
```
p.s I am not an official maintainer. Randomly came across your PR so I thought I could take a look to understand the codebase better..

Reply:
I added that scenario 👍 good call